Formula/ubertooth.rb

```diff
@@ -0,0 +1,26 @@
+class Ubertooth < Formula
+  desc "Host tools for Project Ubertooth"
+  homepage "https://github.com/greatscottgadgets/ubertooth/wiki"
+  url "https://github.com/greatscottgadgets/ubertooth/releases/download/2015-10-R1/ubertooth-2015-10-R1.tar.xz"
+  version "2015-10-R1"
+  sha256 "bc37e7978d137a64d918d7c8f1e7ca9cff093f9921d805e9809b12e5ab12ae35"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libbtbb"
+  depends_on "libusb"
+
+  def install
+    chdir("host") do
+      mkdir "build" do
+        system "cmake", "..", *std_cmake_args
+        system "make", "install"
+      end
+    end
+  end
+
+  test do
+    # Most ubertooth utilities require an ubertooth device present, ubertooth-rx with the -i option does not.
```

Comment:
Can probably squish this notation a bit. Something like:
```ruby
# Most ubertooth utilities require an ubertooth device present.
```
Alone should be fine. It could be inferred from that this test is the exception to the rule there.